content/docs/enterprise/changelog.mdx

```diff
@@ -15,6 +15,34 @@ import Versioning from '../admonitions/_semantic-versioning.mdx';
 
 :::
 
+## 0.27.0
+
+### Improvements
+
+- The Enterprise Console now includes a “Report issue” feedback widget, allowing you to easily report any problems you may encounter. We look forward to hearing your feedback!
+
+  This widget is loaded from a third-party service. If you do not wish to include this on your Enterprise Console installation, you can disable the feedback widget by setting the new option `--disable-feedback-widget`.
```

Comment:
Might be nice to link to the configuration page here once that is updated to document this option.

Reply:
Done.